tips/TIP-0032/tip-0032.md

```diff
@@ -0,0 +1,104 @@
+---
+tip: 32
+title: Shimmer Protocol Parameters
+description: Describes the global protocol parameters for the Shimmer protocol
+author: Wolfgang Welz (@Wollac) <wolfgang.welz@iota.org>, Levente Pap (@lzpap) <levente.pap@iota.org>
+discussions-to: https://github.com/iotaledger/tips/pull/71
+status: Draft
+type: Standards
+layer: Core
+created: 2022-04-04
+---
+
+# Summary
+
+This TIP describes the global protocol parameters for the Shimmer protocol.
+
+# Motivation
+
+Various other protocol TIPs rely on certain constants that need to be defined for an actual implementations of nodes or other applications using the protocol. This TIP serves as a single document to provide these parameters. It also serves as a historical record of protocol parameter changes.
+
+# Detailed design
+
+## Shimmer Units
+
+The whole Shimmer genesis token supply has been distributed to IOTA token holders who staked their tokens to receive SMR during
+the Shimmer staking event:
+ - Start date: Tuesday, December 28, 2021 3:00 PM CET ([Milestone #2102114](https://explorer.iota.org/mainnet/milestone/2102114))
+ - End date: Monday, March 28, 2022 8:12 PM CET ([Milestone #2879714](https://explorer.iota.org/mainnet/milestone/2879714))
```

Comment:
Maybe only me but these links show nothing ?

Reply:
@Thoralf-M solved it :)